docs/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Do we need a separate `package.json` for the docs? On the other projects we add dev dependencies and scripts for building/previewing the docs. Probably simpler.

Reply:
I was assuming we’d have a separate `package.json` and update the build script. I think they need to be separate as a package can’t depend on itself?